lib/mailer/sfMailer.class.php

```diff
@@ -282,7 +283,7 @@ public function sendNextImmediately()
    *
    * @return int|false The number of sent emails
    */
-  public function send(Swift_Mime_Message $message, &$failedRecipients = null)
+  public function send($message, &$failedRecipients = null)
```

Comment:
[minor] It will be a good opportunity to fix the PHPDoc of the first argument as it is not a `Swift_Transport` but a message. So document what type to pass depending on Swift Mailer version.

Reply:
I've added both allowed classes as `Swift_Mime_Message|Swift_Mime_SimpleMessage` in the docblock. Is that ok or would you prefer a typeless message and having the correct type explained in the description?

Reply:
That's the minimum things to do in term of documentation as on each cases it works.

`Swift_Mime_Message` is not available on SwiftMailer version 6 then cannot be instantiate.